core/src/main/java/io/grpc/internal/ServerCallImpl.java

```diff
@@ -205,7 +205,7 @@ public String getAuthority() {
    * on.
    */
   private void internalClose(Status internalError) {
-    stream.close(internalError, new Metadata());
+    stream.cancel(internalError);
```

Comment:
This status is stored internally in gRPC, but won't actually be exposed anywhere on server-side. We should probably go ahead and log it here.

Reply:
ok. which level? Is SEVERE the correct level?

Reply:
WARNING seems good. SEVERE seems a bit too much, because we were able to recover.